Docker-Swarm-deployment/single-node/immudb/example-secrets/secrets/config.json

```diff
@@ -1,59 +1,42 @@
-            "indexing_on": "auditingtable(userID, iudxID, time, api)",
-            "flush_threshold": 600,
-            "sync_threshold": 600,
-            "cleanup_percentage": 0.1
-          },
-          {
-            "database_name": "iudxrsorg",
-            "table": "rsaudit(id VARCHAR[128] NOT NULL,api VARCHAR[128] NOT NULL,userid VARCHAR[128] NOT NULL,epochtime INTEGER NOT NULL,resourceid VARCHAR[256] NOT NULL,isotime VARCHAR[64] NOT NULL,providerid VARCHAR[128] NOT NULL,size INTEGER, PRIMARY KEY id)",
-            "indexing_on": "rsaudit(userid, epochtime, providerid, resourceid)",
-            "flush_threshold": 1000,
-            "sync_threshold": 1000,
-            "cleanup_percentage": 0.1
-          }
-    ],
-
-    "users": [
-
-        {
-            "username": "iudx_auth",
-            "password": "/run/secrets/password/auth-password",
-            "database_name": "iudxauth"
-          },
-          {
-            "username": "iudx_cat",
-            "password": "/run/secrets/password/cat-password",
-            "database_name": "iudxcat"
-          },
-          {
-            "username": "iudx_rs",
-            "password": "/run/secrets/password/rs-password",
-            "database_name": "iudxrsorg"
-        }
-    ]
-
-
-
+{
+  "immudb_host": "immudb",
+  "immudb_default_user": "immudb",
+  "immudb_default_user_password": "immudb",
+  "change_admin_password": true,
+  "admin_password": "/run/secrets/password/admin-password",
+  
+  
+  "database": "iudx",
+  "flush_threshold": 1000,
+  "sync_threshold": 1000,
+  "cleanup_percentage": 0.1,
+  
+  "tables": [
+  {
+    "table": "auditing_acl_apd( id VARCHAR[128] NOT NULL, userid VARCHAR[128] NOT NULL, endpoint VARCHAR[128] NOT NULL, method VARCHAR[128] NOT NULL, body VARCHAR[128] NOT NULL,size INTEGER NOT NULL,isotime VARCHAR[128] NOT NULL, epochtime INTEGER NOT NULL, PRIMARY KEY id)",
+    "indexing_on": "auditing_acl_apd(userid,endpoint,epochtime)"
+  },
+  {
+    "table": "auditingtable(id VARCHAR[128] NOT NULL, userRole VARCHAR[64] NOT NULL,userID VARCHAR[128] NOT NULL,iid VARCHAR[250] NOT NULL,api VARCHAR[128] NOT NULL,method VARCHAR[32] NOT NULL,time INTEGER NOT NULL,iudxID VARCHAR[256] NOT NULL,PRIMARY KEY id)",
+    "indexing_on": "auditingtable(userID, iudxID, time, api)"
+  },
+  {
+    "table": "rsaudit(id VARCHAR[128] NOT NULL,api VARCHAR[128] NOT NULL,userid VARCHAR[128] NOT NULL,epochtime INTEGER NOT NULL,resourceid VARCHAR[256] NOT NULL,isotime VARCHAR[64] NOT NULL,providerid VARCHAR[128] NOT NULL,size INTEGER, PRIMARY KEY id)",
+    "indexing_on": "rsaudit(userid, epochtime, providerid, resourceid)"
   }
-  
\ No newline at end of file
+],
+"users": [
+  {
+    "username": "auditing_user",
+    "password": "/run/secrets/password/auditing-password",
```

Comment:
change create_secrets password accordingly in K8s and docker swarm. Also change example config in K8s